Dockerfile.pipelines

```diff
@@ -93,10 +76,10 @@ RUN chown -R devito /home/devito
 USER devito
 WORKDIR /home/devito
 
-RUN if [ $installWithPip == "true" ] ; then \ 
-      python setup.py install --user ; \
+RUN if [ $installWithPip == "true" ] ; then \
+      python3 setup.py install --user ; \
```

Comment:
why isn't ``python`` the default ``python3``?

Reply:
default is not the 3.6 which is called python3